### PR TITLE
Cleanup hack/ Dockerfiles

### PR DESCRIPTION
**Pull request by jessfraz**
``` release-note
Small improvements to hack/ Dockerfiles
```

[Analytics]()

### CODE REVIEW

**Comment by spxtr** [comment]
ok to test

File: hack/gen-swagger-doc/Dockerfile [hunk 1]
```diff
@@ -14,27 +14,32 @@
...
-RUN wget https://services.gradle.org/distributions/gradle-2.5-bin.zip
-RUN mkdir build/
-RUN unzip gradle-2.5-bin.zip -d build/
-
-RUN mkdir gradle-cache/
+RUN apt-get update && apt-get install -y \
+	asciidoctor \
+	unzip \
+	--no-install-recommends \
+	&& rm -rf /var/lib/apt/lists/*
```
Comment (dims) [inline hack/gen-swagger-doc/Dockerfile]: Nice cleanup of the downloaded stuff (rm -rf). Had not seen this before. Thanks! (http://askubuntu.com/questions/179955/var-lib-apt-lists-huge-in-12-04)

**Comment by spxtr** [comment]
Thanks for this :). At some point I saw that `apt-get` misuse and meant to make a fix, but I forgot.

**Comment by spxtr** [comment]
I think you'll want to push the new image to gcr.io and bump the version.

**Comment by jfrazelle** [comment]
oooo I don't think I have permissions to do that

On Wed, Jun 22, 2016 at 6:23 PM, Joe Finney <notifications@github.com>
wrote:

> I think you'll want to push the new image to gcr.io and bump the version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27901#issuecomment-227924668>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABYNbFfGHiyFRCEdgA7Jx1WwEcDO4jfkks5qOeAOgaJpZM4I8Qam>
> .
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by spxtr** [comment]
ok to test

@jfrazelle what's the status of this PR?

**Comment by jfrazelle** [comment]
I bumped the image tags and pushed them as well, eeeeek hopefully its all good

**Pull request merged**